dev/com.ibm.ws.concurrent.persistent/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -41,7 +41,7 @@ jndiName=JNDI name
 jndiName.desc=JNDI name.
 
 missedTaskThreshold=Missed task threshold for fail over
-missedTaskThreshold.desc=Amount of time beyond the expected start of a task execution to reserve for running it. Other members are prevented from running the task before the expiration of this interval. If the interval elapses without successful execution of the task, then the task execution is considered missed, enabling another member to attempt to run it. This enables fail over.
+missedTaskThreshold.desc=Amount of time beyond the expected start of a task execution to reserve for running it. Other members are prevented from running the task before the expiration of this interval. If the interval elapses without successful execution of the task or the task rolls back, then the task execution is considered missed, enabling another member to attempt to run it. Missed task threshold values within the supported range of 100 seconds to 9000 seconds (2.5 hours) enable failover.
```

Comment:
**change**
Amount of time beyond the expected start of a task execution to reserve for running it.
**to**
The amount of time after the expected start of a task execution that is reserved for running the task.

Reply:
The fix for the above has been made exactly as suggested under https://github.com/OpenLiberty/open-liberty/pull/10521/commits/9d01e544ea48cebdf0ae90ef0787d6af15ef4e25